pages/sdk-protocol-kit/guides/safe-signatures.md

```diff
@@ -0,0 +1,844 @@
+| Hex              | 1     | 0x                                                                                                                                                                                                                                                                   | Hex string characters                                                                                                                            |
+| Verifier         | 32    | 000000000000000000000000f75D61D6C27a7CC5788E633c1FC130f0F4a62D33                                                                                                                                                                                                     | Padded address of the contract that implements the EIP-1271 interface to verify the signature. The Safe signer address                           |
+| Data position    | 32    | 0000000000000000000000000000000000000000000000000000000000000041                                                                                                                                                                                                     | Start position of the signature data (offset relative to the beginning of the signature data). 41 hex is 65 in decimal                           |
+| Signature Type   | 1     | 00                                                                                                                                                                                                                                                                   | 00 for [Safe Accounts](https://github.com/safe-global/safe-contracts/blob/f03dfae65fd1d085224b00a10755c509a4eaacfe/contracts/Safe.sol#L322-L336) |
+| Signature Length | 32    | 0000000000000000000000000000000000000000000000000000000000000082                                                                                                                                                                                                     | The length of the signature. 82 hex is 130 in decimal                                                                                            |
+| Signature        | 130   | 023d1746ed548e90f387a6b8ddba26e6b80a78d5bfbc36e5bfcbfd63e136f8071db6e91c037fa36bde72159138bbb74fc359b35eb515e276a7c0547d5eaa042520d3e6565e5590641db447277243cf24711dce533cfcaaf3a64415dcb9fa309fbf2de1ae4709c6450752acc0d45e01b67b55379bdf4e3dc32b2d89ad0a60c231d61f | Signature bytes that are verified by the signature verifier (130 bytes are represented by 260 characters in an hex string)                       |
+
+The decomposition appears unchanged, but there are two changes: the signature length has doubled because there are now two signatures, and the signature itself is a concatenation of the two regular signatures.
```

Comment:
```suggestion
The table looks very similar to the previous one, but there are two changes:

- The _signature length_ has doubled because this Safe needs two signatures
- The _signature_ value is a concatenation of the two regular signatures.

```